spec/rswag/v0/form214192_spec.rb

```diff
@@ -0,0 +1,184 @@
+              employerName: { type: :string },
+              employerAddress: {
+                type: :object,
+                properties: {
+                  street: { type: :string, example: '456 Business Blvd' },
+                  street2: { type: :string, example: 'Suite 200' },
+                  city: { type: :string, example: 'Chicago' },
+                  state: { type: :string, example: 'IL' },
+                  postalCode: { type: :string, example: '60601' },
+                  country: { type: :string, example: 'USA' }
+                },
+                required: %i[street city state postalCode country]
+              },
+              typeOfWorkPerformed: { type: :string },
+              beginningDateOfEmployment: { type: :string, format: :date },
+              endingDateOfEmployment: { type: :string, format: :date },
+              amountEarnedLast12MonthsOfEmployment: { type: :number },
+              timeLostLast12MonthsOfEmployment: { type: :string },
+              hoursWorkedDaily: { type: :number },
+              hoursWorkedWeekly: { type: :number },
+              concessions: { type: :string },
+              terminationReason: { type: :string },
+              dateLastWorked: { type: :string, format: :date },
+              lastPaymentDate: { type: :string, format: :date },
+              lastPaymentGrossAmount: { type: :number },
+              lumpSumPaymentMade: { type: :boolean },
+              grossAmountPaid: { type: :number },
+              datePaid: { type: :string, format: :date }
+            }
+          },
+          militaryDutyStatus: {
+            type: :object,
+            properties: {
+              currentDutyStatus: { type: :string },
+              veteranDisabilitiesPreventMilitaryDuties: { type: :boolean }
+            }
+          },
+          benefitEntitlementPayments: {
+            type: :object,
+            properties: {
+              sickRetirementOtherBenefits: { type: :boolean },
+              typeOfBenefit: { type: :string },
+              grossMonthlyAmountOfBenefit: { type: :number },
+              dateBenefitBegan: { type: :string, format: :date },
+              dateFirstPaymentIssued: { type: :string, format: :date },
+              dateBenefitWillStop: { type: :string, format: :date },
+              remarks: { type: :string }
+            }
+          }
+        }
+      }
+
+      # Success response
+      response '200', 'Form successfully submitted' do
+        schema type: :object,
+               properties: {
+                 data: {
+                   type: :object,
+                   properties: {
+                     id: { type: :string },
+                     type: { type: :string },
+                     attributes: {
+                       type: :object,
+                       properties: {
+                         submitted_at: { type: :string, format: 'date-time' },
+                         regional_office: { type: :array },
```

Comment:
```suggestion
                         regional_office: { type: :array,  items: { type: 'string' } },
                         
                         
```

Reply:
Nice. I'll add that and throw in an example as well